rever.xsh

```diff
@@ -1,14 +1,18 @@
 $PROJECT = 'rever'
 $REVER_DIR = 'rever-tmp'
-$ACTIVITIES = ['pytest', 'version_bump', 'changelog', 'tag']
+$ACTIVITIES = ['pytest', 'version_bump', 'changelog', 'tag', 'conda_forge']
 
 $VERSION_BUMP_PATTERNS = [
     ('rever/__init__.py', '__version__\s*=.*', "__version__ = '$VERSION'"),
     ('setup.py', '        version\s*=.*,', "        version='$VERSION',")
     ]
 $CHANGELOG_FILENAME = 'CHANGELOG.rst'
 $CHANGELOG_IGNORE = ['TEMPLATE.rst']
-$TAG_REMOTE = 'git@github.com:ergs/rever.git'
+$TAG_REMOTE = 'git@github.com:regro/rever.git'
+
+$GITHUB_ORG = 'regro'
+$GITHUB_REPO = 'rever'
+$GITHUB_CREDFILE = '~/.gh_cred.txt'
```

Comment:
This should still be removed...

Reply:
Terribly sorry, I think I got it this time.